config/districts.yml

```diff
@@ -1,2 +1,30 @@
-- Eixample
-- Ciutat Vella
+- 
+  - Ciutat Vella
```

Comment:
Well, I used an array of arrays in order to pass it directly to the form select helper. I can try your approach but is it for readability or something else?